README.md

```diff
@@ -1,2 +1,81 @@
-# handbook-of-best-practices
-Software Quality Assurance Assignment
+# Handbook of Best Practices
+
+## Task Estimation
+
+
+The Sprint Planning meeting is the Scrum ceremony where the team estimates all tasks associated with each user story in the Sprint backlog. 
+The estimation effort focuses on all resources required to complete the tasks for each Sprint. 
+
+A shared task list is used to estimate the 
+duration and effort for each user story in the backlog. This practice results in a shared team perspective, resulting in a task estimation 
+process that is consistent and reliable throughout all Sprints. 
+In this section of the handbook we will discuss the following points : 
+
+
+ + Building a successful story point estimation
+ + The importence of story point estimation
+ + Challenges of accurately estimating tasks
+ + Overcoming estimation challenges
+ + Bad practices of task estimation
+ + Advantages and disadvantages of twice estimation
+ 
+
+&nbsp;
+
+
+### 1. Building a successful story point estimation
+
+---
+In Task Estimation, team does a comparative analysis between all of the stories for the project. For each story to be sized, 
+ the following as a team(Product Owner, Core Scrum team including developers, testers & scrum master):
+
+
+
+&nbsp;
+
+<img src="https://github.com/xtreger/handbook-of-best-practices/blob/task-estimation-in-scrum-jouhara/Images/steps.png" width="500" height="200"/>
+
+&nbsp;
+
+### 2. The importence of story point estimation
```

Comment:
Can you correct the spelling of importance?

Reply:
Done